backend/service_accounts/scopes.py

```
@@ -6,4 +6,10 @@
     AI_REVIEW_PROPOSE_SCOPE,
 )
 
-ALLOWED_SERVICE_ACCOUNT_SCOPES = frozenset(AI_REVIEW_SCOPES)
+# Granted to the Deckard Telegram support bot so it can redeem validator
+# Telegram group bind codes (validators app).
+TELEGRAM_BIND_REDEEM_SCOPE = 'telegram_bind:redeem'
+
+ALLOWED_SERVICE_ACCOUNT_SCOPES = frozenset(
+    AI_REVIEW_SCOPES + (TELEGRAM_BIND_REDEEM_SCOPE,)
```

Ruff (0.16.0)

[warning] Consider `(*AI_REVIEW_SCOPES, TELEGRAM_BIND_REDEEM_SCOPE)` instead of concatenation

Replace with `(*AI_REVIEW_SCOPES, TELEGRAM_BIND_REDEEM_SCOPE)`

(RUF005)